lib/selbot2/cla.rb

```diff
@@ -0,0 +1,27 @@
+require 'uri'
+
+module Selbot2
+  class CLA
+    include Cinch::Plugin
+
+    prefix Selbot2::PREFIX
+    match /(?:cla) (.+)/
+
+
+    def execute(message, query)
+      commits_url = "https://github.com/SeleniumHQ/selenium/commits?author=#{URI.escape query}"
+      resp   = RestClient.get(commits_url)
+      result = Nokogiri.XML(resp)
+
+      if result.css('.commit').any?
+        message.reply "#{query} has signed the CLA and has #{result.css('.commit').size} commits."
```

Comment:
this isn't quite accurate, since we don't really know that... since we're not looking at the cla and it assumes that someone verified they signed it (only one person can do that currently).  Maybe just change the verbage to say "#{query} has had #{result.css('.commit').size} commits merged"

and the negative saying "has had no commits merged"